compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -1108,6 +1110,9 @@ class Definitions {
    *  is read from a classfile.
    */
   def patchStdLibClass(denot: ClassDenotation)(using Context): Unit =
+    val annot = Annotation.deferredSymAndTree(defn.CompileTimeOnlyAnnot) {
+      Literal(Constant("The symbol can only be used at compile time in import statements"))
```

Comment:
I suggest putting those annotations directly in the source code in stdlibPatches/language.scala, because they might not apply to other definitions (like those in stdlibPatches/Predef.scala or others we may add in the future)